src/modules/coinmarket/CoinGeckoMarket.ts

```diff
@@ -48,7 +50,8 @@ export class CoinGeckoMarket implements CoinMarket {
                     resolve(true);
                 }
             }).catch((err: any) => {
-                logger.error(`Stoa is unable to ping gecko coin market`);
+                logger.error(`Stoa is unable to ping gecko coin market`,
+                 { operation: Operation.connection, height: Stoa.height.toString(), success: false });
```

Comment:
Stoa.height.toString() There is no need to display here. You only need to display it in relation to the block.